modules/integration_aws-elasticsearch/conf/02-4xx.yaml

```diff
@@ -0,0 +1,32 @@
+module: AWS Elasticsearch
+name: 4xx HTTP response
+id: fourxx_http_response
+
+aggregation: true
+filtering: "filter('namespace', 'AWS/ES') and filter('stat', 'sum')"
+tip: "In addition to the runbook, have a look at https://aws.amazon.com/premiumsupport/knowledge-center/opensearch-resolve-429-error/ and  https://docs.aws.amazon.com/opensearch-service/latest/developerguide/handling-errors.html#troubleshooting-throttle-api"
+runbook_url: "https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-cloudwatchmetrics.html"
+value_unit: "%"
+
+signals:
+  A:
+    metric: 4xx
+  B:
+    metric: 2xx
+  signal:
+    # "OpenSearchRequests" can be inferiore to 4xx (I don't know how), it's not the sum of 2xx, 3xx, 4xx and 5xx… so I juste compute
```

Comment:
I've posted my understanding of this behavior here: https://stackoverflow.com/a/72818868/316805

TL;DR: `4xx` cannot be compared to `OpenSearchRequests`, but `5xx` can.

Reply:
Finally, I believe it is not worth monitoring `4xx` requests as they include unauthorized accesses ending up in `403` HTTP status codes.

Reply:
disabled + note in the readme